tests/models/owlvit/test_modeling_owlvit.py

```diff
@@ -120,7 +120,7 @@ def create_and_check_model(self, config, pixel_values):
         # expected sequence length = num_patches + 1 (we add 1 for the [CLS] token)
         num_patches = (self.image_size // self.patch_size) ** 2
         self.parent.assertEqual(result.last_hidden_state.shape, (self.batch_size, num_patches + 1, self.hidden_size))
-        self.parent.assertEqual(result.pooler_output.shape, (self.batch_size, num_patches + 1, self.hidden_size))
+        self.parent.assertEqual(result.pooler_output.shape, (self.batch_size, self.hidden_size))
```

Comment:
Hi @alaradirik, after removing the `use_hidden_state` param, the pooled output shape has changed. Is this commit okay?

Reply:
Hey @unography yes, the commit is ok but the relevant tests will need to updated. 

The test is failing because we were previously returning modified or unmodified vision embeddings and pooler_output based on where the vision model was called (base model or model head). We are now always returning unmodified CLIP/OwlViTModel embeddings as the pooler_output and additionally generate modified embeddings if needed.

Reply:
sure, I'll check the test cases. The other issues should be resolved now

Reply:
@unography could you ping me when you want me to do the final review? I'll be adding a core maintainer to approve the changes as well